<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditActionDialog</class>
 <widget class="QDialog" name="EditActionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Action</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <property name="horizontalSpacing">
      <number>6</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="shortcut_L">
       <property name="text">
        <string>&amp;Shortcut:</string>
       </property>
       <property name="buddy">
        <cstring>shortcut_SS</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="ShortcutSelector" name="shortcut_SS">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="description_L">
       <property name="text">
        <string>&amp;Description:</string>
       </property>
       <property name="buddy">
        <cstring>description_LE</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="description_LE"/>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="enabled_CB">
       <property name="text">
        <string>&amp;Enabled</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QRadioButton" name="command_RB">
         <property name="text">
          <string>&amp;Command</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="dbus_method_RB">
         <property name="text">
          <string>&amp;DBus message</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="type_L">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QStackedWidget" name="action_SW">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="command_P">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="command_L">
           <property name="text">
            <string>Co&amp;mmand:</string>
           </property>
           <property name="buddy">
            <cstring>command_PTE</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPlainTextEdit" name="command_PTE"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="dbus_method_P">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="service_L">
           <property name="text">
            <string>S&amp;ervice:</string>
           </property>
           <property name="buddy">
            <cstring>service_LE</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="service_LE"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="path_L">
           <property name="text">
            <string>&amp;Path:</string>
           </property>
           <property name="buddy">
            <cstring>path_LE</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="path_LE"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="interface_L">
           <property name="text">
            <string>&amp;Interface:</string>
           </property>
           <property name="buddy">
            <cstring>interface_LE</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="interface_LE"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="method_L">
           <property name="text">
            <string>&amp;Method:</string>
           </property>
           <property name="buddy">
            <cstring>method_LE</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="method_LE"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox_1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ShortcutSelector</class>
   <extends>QToolButton</extends>
   <header>shortcut_selector.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>shortcut_SS</tabstop>
  <tabstop>description_LE</tabstop>
  <tabstop>enabled_CB</tabstop>
  <tabstop>command_RB</tabstop>
  <tabstop>dbus_method_RB</tabstop>
  <tabstop>command_PTE</tabstop>
  <tabstop>service_LE</tabstop>
  <tabstop>path_LE</tabstop>
  <tabstop>interface_LE</tabstop>
  <tabstop>method_LE</tabstop>
  <tabstop>buttonBox_1</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_1</sender>
   <signal>accepted()</signal>
   <receiver>EditActionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_1</sender>
   <signal>rejected()</signal>
   <receiver>EditActionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
